x-pack/metricbeat/module/panw/routing/_meta/fields.yml

```diff
@@ -0,0 +1,150 @@
+- name: routing
+  type: group
+  release: beta
+  description: >
+    routing
+  fields:
+    - name: bgp.peer_name
+      type: keyword
+      description: >
+        The name of the current peer in the BGP peer group
+    - name: bgp.virtual_router
+      type: keyword
+      description: >
+        The virtual router with which the BGP peer is associated
+    - name: bgp.peer_group
+      type: keyword
+      description: >
+        The name of the BGP peer group this peer belongs to
+    - name: bgp.peer_router_id
+      type: ip
+      description: >
+        BGP peer router ID
+    - name: bgp.remote_as_asn
+      type: long
+      description: >
+        The remote Autonomous System (AS) number of the peer
+    - name: bgp.status
+      type: keyword
+      description: >
+        The BGP session status, e.g., "Established" means the session is up and running
+    - name: bgp.status_duration
+      type: long
+      description: >
+        Time in seconds since the current status was set
+    - name: bgp.password_set
+      type: boolean
+      description: >
+        Indicates whether a password is set for the BGP peer
+    - name: bgp.passive
+      type: boolean
+      description: >
+        Indicates if the BGP peer is in passive mode: if yes then router will not initiate a connection to the peer
+    - name: bgp.multi_hop_ttl
+      type: long
+      description: >
+        Time to Live (TTL) value for multi-hop BGP sessions.
+    - name: bgp.peer_address
```

Comment:
@tommyers-elastic 

Should we consider running an ingest pipeline to extract the values and keep the extracted field value as values of field of type `ip` and `integer`?